src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -48,6 +48,12 @@ static RuntimeMetrics()
             () => GetGarbageCollectionCounts(),
             description: "Number of garbage collections that have occurred since process start.");
 
+        MeterInstance.CreateObservableUpDownCounter(
+            "process.runtime.dotnet.gc.heap.total_objects_size",
```

Comment:
This is very inconsistent with other metrics names (e.g. https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/005fdb71b709d5eefbc8c5b7fa5a2c3f14fddecf/src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs#L53) in the existing library. Consider:

```suggestion
            "process.runtime.dotnet.gc.live_objects.size",
```

Reply:
renamed in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/683/commits/25e9cf80f5cffbd2e81960003220bc0445ef7632

Reply:
'live' usually means that an object is reachable via references from a root, but that isn't necessarily true for these objects at the current instant in time. We know these objects haven't been collected yet, but they may already be unreachable. We can brainstorm different names but I would avoid "live". For example "process.runtime.dotnet.gc.objects.size"?

Reply:
> 'live' usually means that an object is reachable via references from a root, but that isn't necessarily true for these objects at the current instant in time. We know these objects haven't been collected yet, but they may already be unreachable. We can brainstorm different names but I would avoid "live". For example "process.runtime.dotnet.gc.objects.size"?

Sounds good to me, and the name "process.runtime.dotnet.gc.objects.size" is aligned with the existing ones.

Reply:
renamed in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/683/commits/48e021b49143b68fa59ad9f95c677b72ee71ca78